test/markdown/markdown_test.py

```diff
@@ -0,0 +1,49 @@
+import io
+import os
+import sys
+import yaml
+
+ROOT_DIR = '/email_markdown_files/us'
+
+WHITELISTED_KEYS = [
```

Comment:
lol this is pretty ironic that it's here of all places, but at least at Google the style guide would say this should be ALLOWLISTED_KEYS

Reply:
I'm a firm believe in goog style guides - changed!